src/components/webView.re

```diff
@@ -1,15 +1,22 @@
 [@bs.module "react-native"] external view : ReasonReact.reactClass = "WebView";
 
-type iOSLoadRequestEvent = {
+type navigationType = [
+  | [@bs.as "click"] `click
+  | [@bs.as "formsubmit"] `formsubmit
+  | [@bs.as "backforward"] `backforward
+  | [@bs.as "reload"] `reload
+  | [@bs.as "formresubmit"] `formresubmit
+  | [@bs.as "other"] `other
```

Comment:
The warnings you're seeing in refmt are because these `@bs.as` are not used without `[@bs.deriving jsConverter]` on the type.